rollup.config.js

```diff
@@ -1,9 +1,25 @@
-export default {
+import { terser } from "rollup-plugin-terser";
+
+// Declaration required to apply JSDoc type tag
+/** @type {import("rollup").RollupWatchOptions} */
+const opts = {
 	input: 'mgrs.js',
-  output: {
+  output: [{
```

Comment:
Is it possible to export a non-minified bundle, in addition to `mgrs.min.js` and `mgrs.esm.js`?  I understand that the sourcemaps should work for the minified bundle, but I can see it being useful to people for debugging purposes to have an unminified output/bundle.

Reply:
Yes, you just pass an `include` to the terser plugin like [in the docs](https://github.com/TrySound/rollup-plugin-terser#includeexclude) -- if the output file name doesn't match one of the `include` expressions then terser won't touch it.

Reply:
I'll work on this today.